src/tags/index.js

```diff
@@ -105,8 +105,10 @@ module.exports = function(fractal){
                             if (name.indexOf('attributes') > -1) {
                                 innerContext[name].merge(value);
                             }
-                            else {
-                                innerContext[name] = value;
+                            // If the parent context key is missing (undefined) or true,
+                            // render the component default value, use passed value otherwise.
```

Comment:
I added a fixup commit addressing your comments.

1. Changed. You're right, wasn't clear enough

2. I checked this again (as it wasn't clear to me neither, but it worked 🤷‍♂) and wondered whether it will fail if we don't pass the key at all but it didn't. I addressed this anyway and rewrote the code as you mentioned in 3. to make it more clear.

3. see 2.

4. No, as we strictly check for undefined. Setting a value to null will render an empty value